Neo.css

```diff
@@ -1,94 +1,197 @@
 }
 
+
+
+
 ::placeholder {
-  /* Chrome, Firefox, Opera, Safari 10.1+ */
   color: crimson;
+  text-align: center;
+  text-shadow: 0 0 7px red, 0 0 11px red;
+  
 }
 
+
 input {
   background-color: rgb(0, 0, 0);
   color: crimson;
+  font-family: 'Courier Prime', monospace;
+  box-sizing: border-box;
+  padding-top: 3%;
+  padding-bottom: 3%;
+  width: 70%;
+  text-shadow: 0 0 7px red, 0 0 11px red;
+  
+} 
+
+.add-task-btn {
+  white-space: nowrap;
+  width: 30%;
+  margin: 0;
 }
 
+
+
 button {
   background-color: teal;
-  margin: 10px;
+  font-family: 'Courier Prime', monospace;
+  margin: 0.7%;
 }
 
-h1 {
-  color: rgb(255, 176, 0);
-  font-family: "White Rabbit";
+.not-done-header {
+  color: #f0fff8;;
+  font-family: 'Courier Prime', monospace;
   font-weight: 100;
-  text-align: center;
-  align-self: center;
+  overflow: auto;
+  text-shadow: 0 0 1px #80ffc0, 0 0 3px #00ff66, 0 0 7px #00ff66, 0 0 11px #00ff66;
+  /* display: inline-table; */
+  padding: 5%;
 }
 
-h2 {
-  color: rgb(255, 176, 0);
-  text-align: center;
-  align-self: center;
-}
-
-@font-face {
-  font-family: "White Rabbit";
-  src: url("//db.onlinewebfonts.com/t/0253e1401c58c5911902cdca02857f0e.eot");
-  src: url("//db.onlinewebfonts.com/t/0253e1401c58c5911902cdca02857f0e.eot?#iefix")
-      format("embedded-opentype"),
-    url("//db.onlinewebfonts.com/t/0253e1401c58c5911902cdca02857f0e.woff2")
-      format("woff2"),
-    url("//db.onlinewebfonts.com/t/0253e1401c58c5911902cdca02857f0e.woff")
-      format("woff"),
-    url("//db.onlinewebfonts.com/t/0253e1401c58c5911902cdca02857f0e.ttf")
-      format("truetype"),
-    url("//db.onlinewebfonts.com/t/0253e1401c58c5911902cdca02857f0e.svg#White Rabbit")
-      format("svg");
+ .done-header {
+  color: #f0fff8;;
+  font-weight: 100;
+  overflow: auto;
+  text-shadow: 0 0 1px rgb(255, 215, 128), 0 0 3px rgb(255, 176, 0), 0 0 7px rgb(255, 176, 0), 0 0 11px rgb(255, 176, 0);
+  /* display: inline-table; */
+  padding: 5%;
+}
+
+ul {
+  overflow: auto;
+  /* display: inline-table; */
+}
+
+
+
+
+
+
+@media (max-width: 347px) { 
+  .addTask {
+    flex-direction: column;
+    align-items: center;
+  }
+  .add-task-btn {
+    width: 70%;
+    border-style: groove;
+  }
+}
+
+@media (min-width: 348px) and (max-width: 402px) {
+  .add-task-btn {
+    width: 79.9531px;
+  }
+}
+
+@media (min-width: 1024px) {
+  .not-done-container {
+    flex-direction: column;
+  }
+  .done-container {
+    flex-direction: column;
+  }
+}
+
+
+
+/* FONT SIZE CHANGES: */
+/* @media (max-width: 380px) {
+  .add-task-btn {
+    font-size: 0.7rem;
+  }
+} */
+@media (min-width: 381px) {
+  input {
+    font-size: 1rem;
+  }
+  ::placeholder {
+    font-size: 1rem;
+  } 
+} 
+@media (min-width: 1024px) {
+  .not-done-header {
+    font-size: 3rem;
+  }
+  .done-header {
+    font-size: 3rem;
+  }
+}
+@media (min-width: 1024px) {
+  ul {
```

Comment:
can we move this to the `min-width: 1024px` media query above? Then we can get rid of this duplicate media query mate